eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/ChannelOpenDualFunded.scala

```diff
@@ -455,24 +455,17 @@ trait ChannelOpenDualFunded extends DualFundingHandlers with ErrorHandlers {
               stay() using d.copy(rbfStatus = RbfStatus.RbfAborted) sending TxAbort(d.channelId, InvalidPushAmount(d.channelId, d.remotePushAmount, msg.fundingContribution.toMilliSatoshi).getMessage)
             } else {
               log.info("our peer wants to raise the feerate of the funding transaction (previous={} target={})", d.fundingParams.targetFeerate, msg.feerate)
-              val fundingParams = InteractiveTxParams(
-                d.channelId,
-                d.fundingParams.isInitiator,
-                d.fundingParams.localAmount, // we don't change our funding contribution
-                msg.fundingContribution,
-                d.fundingParams.fundingPubkeyScript,
-                msg.lockTime,
-                d.fundingParams.dustLimit,
-                msg.feerate,
-                d.fundingParams.requireConfirmedInputs,
+              val fundingParams = d.fundingParams.copy(
+                // we don't change our funding contribution
+                remoteAmount = msg.fundingContribution,
+                lockTime = msg.lockTime,
+                targetFeerate = msg.feerate
               )
               val txBuilder = context.spawnAnonymous(InteractiveTxBuilder(
-                remoteNodeId, nodeParams, fundingParams,
-                d.localPushAmount, d.remotePushAmount,
-                d.metaCommitments.params,
-                d.commitments.localCommit.spec.commitTxFeerate,
-                d.commitments.remoteCommit.remotePerCommitmentPoint,
-                d.commitments.remoteNextCommitInfo.toOption.get,
+                nodeParams, fundingParams,
+                commitmentParams = d.metaCommitments.params,
+                purpose = InteractiveTxBuilder.RbfInitialCommitment(d.commitments.common, d.commitments.commitment),
```

Comment:
No more repetition (and no more weird `remoteNextCommitInfo.toOption.get`), we explicitly reuse the `common` object that we already have.